test/jdk/com/sun/net/httpserver/simpleserver/StressDirListings.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @summary Test to stress directory listings
  * @library /test/lib
- * @run testng/othervm/timeout=180 -Dsun.net.httpserver.nodelay=true StressDirListings
+ * @run testng/othervm/timeout=180 StressDirListings
```

Comment:
Hello Varada @varada1110, in https://github.com/openjdk/jdk/pull/17363 this `StressDirListings` test was updated to set the `-Dsun.net.httpserver.nodelay=true` because without it, the test was failing on AIX. The current PR here is proposing to fix the underlying issue in the HttpServer implementation and is rolling back the addition of this system property to this `StressDirListings` test. Could you or someone from your team please build the latest mainline JDK and apply this current PR on top of it and rerun the test(s) to verify that this change is OK?

Reply:
Hi @jaikiran, I have performed jtreg test on AIX and all the tests are passing.

Reply:
Thank you Varada for running the tests.